bucket-version-mismatch/rewrite_timeseries_bucket_version_mismatch.js

```diff
@@ -89,9 +94,32 @@ function runFixBucketVersionMismatchProcedure(collName) {
   }
 }
 
+function checkValidateResForBucketVersionMismatch(validateRes) {
+  return (validateRes.errors.length != 0 &&
+          validateRes.errors.some(x => x.includes('6698300'))) ||
+      (validateRes.warnings.length != 0 &&
+       validateRes.warnings.some(x => x.includes('6698300')));
+}
+
+function checkLogsForBucketVersionMismatch() {
+  const getLogRes = db.adminCommand({getLog: 'global'});
+  if (getLogRes.ok) {
+    return getLogRes.log
+               .filter(
+                   line =>
+                       (line.includes('6698300') &&
```

Comment:
This is sort of unnecessary as a check because we would only have these `errorMsg` in the case that we have the line include 6698300, but I was thinking it might be a helpful for being an initial filter before we even look at pattern matching with the error messages.